packages/frontend/src/components/explore/ExploreCollectors/Offers.tsx

```diff
@@ -1,82 +1,83 @@
+import { useQuery } from '@tanstack/react-query';
+import { getTradeOffers } from '@/api';
 import { IOfferItem, OfferItem } from '@/components/common/OfferItem/OfferItem';
 
 type Props = {
-  offers?: IOfferItem[];
   onProfile?: boolean;
 };
 
-const DUMMY_OFFERS: IOfferItem[] = [
-  {
-    type: 'BUY',
-    status: 'pending',
-    price: 405,
-    createdAt: '15/03/23',
-    createdBy: {
-      userId: 'aibds-52nsio-78',
-      username: 'yourstruly',
-    },
-    tradeOffer: {
-      comicId: 'marv-wyu-90-rs89',
-      title: 'Marvel Wolverine',
-    },
-  },
-  {
-    type: 'EXCHANGE',
-    status: 'pending',
-    price: 45,
-    createdAt: '16/03/23',
-    createdBy: {
-      userId: 'aibds-52nsio-78',
-      username: 'yourstruly',
-    },
-    tradeOffer: {
-      comicId: 'marv-hav-76-ky78',
-      title: 'Marvel Wolverine',
-    },
-  },
-  {
-    type: 'EXCHANGE',
-    status: 'pending',
-    price: 725,
-    createdAt: '01/02/23',
-    createdBy: {
-      userId: 'aibds-52nsio-78',
-      username: 'yourstruly',
-    },
-    tradeOffer: {
-      comicId: 'marv-hav-30-cy42',
-      title: 'Marvel Wolverine',
-    },
-  },
-  {
-    type: 'BUY',
-    status: 'pending',
-    price: 315,
-    createdAt: '25/01/23',
-    createdBy: {
-      userId: 'aibds-52nsio-78',
-      username: 'yourstruly',
-    },
-    tradeOffer: {
-      comicId: 'marv-hav-15-fz44',
-      title: 'Marvel Wolverine',
-    },
-  },
-  {
-    type: 'BUY',
-    status: 'pending',
-    price: 20,
-    createdAt: '12/02/23',
-    createdBy: {
-      userId: 'aibds-52nsio-78',
-      username: 'yourstruly',
-    },
-    tradeOffer: {
-      comicId: 'marv-hav-94-nx93',
-      title: 'Marvel Wolverine',
-    },
-  },
-];
+// const DUMMY_OFFERS: IOfferItem[] = [
```

Comment:
@Pearl can you remove these comments before merging this branch - very easy to do here on GH just go to this file and remove them directly - commit it to update the branch instantly